frontend/src/pages/Students.vue

```diff
@@ -0,0 +1,356 @@
+
```

Comment:
This file should not be in this PR, it's part of the Table PR. Remember to start a new branch from master when making new PRs.